README.md

```diff
@@ -1,10 +1,21 @@
 # xbps-mini-builder
 
 This is the XBPS mini builder, use it when you want to create a mirror
-containing restricted packages that the Void project doesn't build for
-you.  Put this script in a directory on its own and put a
-`packages.list` next to it that contians the packages you want to
-build.  You can also add an `xbps-src.conf` to be used during builds.
+containing restricted packages that the Void project doesn't build for you. Put
+this script in a directory on its own and put a `packages.list` next to it that
+contains the packages you want to build. You can also add an `xbps-src.conf` to
+be used during builds.
 
-Run the script on a cron or with `snooze(1)` once a day to get
-updates, all other tasks are handled for you!
+If you add packages to `packages.list` after you have run `xbps-mini-builder`
+for the first time, you will need to trigger them manually, since it builds only
+packages with updated templates by default:
+
+```
+./xbps-mini-builder <package>
```

Comment:
This needs a warning that you MUST do this as the user that normally runs the script.  Doing otherwise will break the repo.